tools/ci_build/build.py

```diff
@@ -630,23 +634,23 @@ def tensorrt_run_onnx_tests(build_dir, configs, onnx_test_data_dir):
         else:
            exe = os.path.join(cwd, 'onnx_test_runner')
            model_dir = os.path.join(build_dir, "models")
-        cmd_base = ['-e', 'tensorrt', '-j', '1'] 
+        cmd_base = ['-e', 'tensorrt', '-j', '1']
 
         #onnx test
         if os.path.exists(onnx_test_data_dir):
           onnx_test_cmd = cmd_base + [onnx_test_data_dir]
-          run_subprocess([exe] + onnx_test_cmd, cwd=cwd)
+          run_subprocess([exe] + onnx_test_cmd, cwd=cwd, dll_path=dll_path)
 
         # model test
         # TensorRT can run most of the model tests, but only part of them is enabled here to save CI build time.
         if config != 'Debug' and os.path.exists(model_dir):
-          model_dir_opset8 = os.path.join(model_dir, "opset8")          
+          model_dir_opset8 = os.path.join(model_dir, "opset8")
           model_dir_opset8 = glob.glob(os.path.join(model_dir_opset8, "test_*"))
           model_dir_opset10 = os.path.join(model_dir, "opset10")
           model_dir_opset10 = glob.glob(os.path.join(model_dir_opset10, "tf_inception_v1"))
```

Comment:
can we add another opset10 model to test? preferably one of the models we've been focusing on testing using trt7?
how about opset10/faster_rcnn ?